assets/sass/v2/_footer.scss

```diff
@@ -2,10 +2,14 @@
   .auth-footer {
     .link {
       margin-right: 10px;
+      padding: 8px 0;
```

Comment:
can we use rem values instead of px, helps in scaling on a browser-to-browser basis